incubator/storage_demo/README.md

```diff
@@ -0,0 +1,2 @@
+# acsbc_storage
+Storage demos on ACS
```

Comment:
Can we provide a little more description, some bullet points on what the demo achieves>